Altis_Life.Altis/core/shops/fn_chopShopSell.sqf

```diff
@@ -2,30 +2,28 @@
 /*
     File: fn_chopShopSell.sqf
     Author: Bryan "Tonic" Boardwine
+    Modified: Casperento
 
     Description:
     Sells the selected vehicle off.
 */
 disableSerialization;
-private ["_control","_price","_vehicle","_nearVehicles","_price2","_chopable"];
-_control = CONTROL(39400,39402);
-_price = _control lbValue (lbCurSel _control);
-_vehicle = _control lbData (lbCurSel _control);
+
+private _control = CONTROL(39400,39402);
+private _price = _control lbValue (lbCurSel _control);
+private _vehicle = _control lbData (lbCurSel _control);
 _vehicle = call compile format ["%1", _vehicle];
-_chopable = LIFE_SETTINGS(getArray,"chopShop_vehicles");
-_nearVehicles = nearestObjects [getMarkerPos life_chopShop,_chopable,25];
-_vehicle = (_nearVehicles select _vehicle);
+_vehicle = objectFromNetId _vehicle;
 if (isNull _vehicle) exitWith {};
 
 hint localize "STR_Shop_ChopShopSelling";
 life_action_inUse = true;
-_price2 = CASH + _price;
 [0] call SOCK_fnc_updatePartial;
```

Comment:
Not required anymore?

Reply:
Nope

Reply:
(talking about the updatePartial so we're on the same page - can remove that)

Reply:
Yes, sorry my bad